umh-core/pkg/generated/status-message.go

```diff
@@ -0,0 +1,317 @@
+	Message string `json:"message"`
+	State   string `json:"state"`
+}
+
+type Container struct {
+	Architecture string `json:"architecture"`
+	CPU          CPU    `json:"cpu"`
+	Disk         Disk   `json:"disk"`
+	Hwid         string `json:"hwid"`
+	Memory       Memory `json:"memory"`
+}
+
+type CPU struct {
+	Health Health  `json:"health"`
+	Limit  float64 `json:"limit"`
+	Usage  float64 `json:"usage"`
+}
+
+type Disk struct {
+	Health Health  `json:"health"`
+	Limit  float64 `json:"limit"`
+	Usage  float64 `json:"usage"`
+}
+
+type Memory struct {
+	Health Health  `json:"health"`
+	Limit  float64 `json:"limit"`
+	Usage  float64 `json:"usage"`
+}
+
+type Dfc struct {
+	Connections        []Connection `json:"connections,omitempty"`
+	CurrentVersionUUID *string      `json:"currentVersionUUID"`
+	DeploySuccess      bool         `json:"deploySuccess"`
+	Health             *Health      `json:"health"`
+	Metrics            *DFCMetrics  `json:"metrics"`
+	Name               *string      `json:"name"`
+	Type               Type         `json:"type"`
+	UUID               string       `json:"uuid"`
+	DataContract       *string      `json:"dataContract,omitempty"`
+	InputType          *string      `json:"inputType,omitempty"`
+	IsReadOnly         *bool        `json:"isReadOnly,omitempty"`
+	OutputType         *string      `json:"outputType,omitempty"`
+}
+
+type Connection struct {
+	Health  Health  `json:"health"`
+	Host    string  `json:"host"`
+	Latency float64 `json:"latency"`
+	Name    string  `json:"name"`
+	Port    float64 `json:"port"`
+	UUID    string  `json:"uuid"`
+}
+
+type DFCMetrics struct {
+	FailedMessages   float64 `json:"failedMessages"`
+	Throughput       float64 `json:"throughput"`
+	Unprocessable    float64 `json:"unprocessable"`
+	Unprocessable24H float64 `json:"unprocessable24h"`
+}
+
+type EventsTable struct {
+	Bridges         []string    `json:"bridges"`
+	Error           string      `json:"error"`
+	Origin          *string     `json:"origin"`
+	RawKafkaMessage EventKafka  `json:"rawKafkaMessage"`
+	TimestampMS     float64     `json:"timestamp_ms"`
+	UnsTreeID       string      `json:"unsTreeId"`
+	Value           *EventValue `json:"value"`
+}
+
+type EventKafka struct {
+	Destination            []string          `json:"destination"`
+	Headers                map[string]string `json:"headers"`
+	KafkaInsertedTimestamp float64           `json:"kafkaInsertedTimestamp"`
+	Key                    string            `json:"key"`
+	LastPayload            string            `json:"lastPayload"`
+	MessagesPerMinute      float64           `json:"messagesPerMinute"`
+	Origin                 []string          `json:"origin"`
+	Topic                  string            `json:"topic"`
+}
+
+type Latency struct {
+	Avg float64 `json:"avg"`
+	Max float64 `json:"max"`
+	Min float64 `json:"min"`
+	P95 float64 `json:"p95"`
+	P99 float64 `json:"p99"`
+}
+
+type Redpanda struct {
+	Health     Health  `json:"health"`
+	Throughput float64 `json:"throughput"`
```

Comment:
we need throughput for incoming and outgoing.

 I recommend here stating the unit in the name, like ThroughputIncomingInMsgSec. this will prevent a lot of issues about what unit its is...